src/grammars/syntax.md

```diff
@@ -330,6 +330,9 @@ To make a silent rule, precede the left curly bracket `{` with a low line
 silent = _{ ... }
 ```
 
+Rules called from a silent rule are not treated as silent unless they are
```

Comment:
I wasn't sure of the best terminology. Are the following equivalent and, if so, which is preferred?

* rules _called from_ other rules
* _inner_ rules
* _interior_ rules